### PR TITLE
@damassi => Mount auction results app ahead of artist to support stand-alone auction page

### PR DESCRIPTION
**Pull request by mzikherman**
We deep link to standalone pages for auction lots, those are under `/artist`, so this changes the order of app mounting to support that, moving it in front of the Reaction artist app.

### CODE REVIEW

File: src/desktop/apps/auction_lots/index.coffee [hunk 1]
```diff
@@ -10,6 +10,5 @@ app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
-app.get '/artist/:id/auction-results', timeout('25s'), routes.artist
```
Comment (mzikherman) [inline src/desktop/apps/auction_lots/index.coffee]: Due to this being mounted after, this wasn't getting hit, but this is a sign we should clean up code here. We used to have a separate app implementation for auction results, vs the tabbed view we have now.

Now that this app is mounted ahead, we need to remove this.

File: src/desktop/apps/auction_lots/index.coffee [hunk 2]
```diff
@@ -10,6 +10,5 @@ app = module.exports = express()
...
 app.get '/artist/:artist_id/auction-result/:id', timeout('25s'), routes.detail
```
Comment (mzikherman) [inline src/desktop/apps/auction_lots/index.coffee]: Here's that standalone auction result route.

**Comment by mzikherman** [comment]
I tested this locally with https://github.com/artsy/reaction/pull/1339 and `yarn link`, and everything behaved as we wanted (I think?!).

https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-284

Specifically those three examples included.

**Comment by damassi** [comment]
👍 

merge on green

**Pull request merged**